### DISCUSSION TITLE
Is there a chance that u8g2 will support LCM-553-60601

### DISCUSSION THREAD

**Discussion by RaspCla** [olikraus/u8g2#1553] [Ideas]
Hello,
There are lots of older Displays type LCM-553-60601 available.
This display is build with a MSM5238 and 6 MSM5839b circuits. I found the datasheets here:
[mikrocontroller.net LCM-553-60601 LCD Grafikdisplay](https://www.mikrocontroller.net/attachment/1792/lcd.zip)
There is also a information about the display included (lcm_info.pdf) which shows some details of the display:
e.g. 120 x 64 pixel (2x32)

I also found a modified T6963 library, the author statet it was tested with LCM-553-60601 display:
[ ct2034-T6963_Lib](https://github.com/ct2034/T6963_Lib)
I was able to compile the code with the arduino ide but the display is not working and I can even not see any signal at any ports (using an oszilloscope).

So my question is if there is a chance to support the LCM-553-60601 display via u8g2 lib.
I am also willing to support if possible

**Comment by olikraus** [comment]
Almost impossible. U8g2 assumes some kind of display controller, which is not there with your display.
U8g2 supports the T6963, but I can not image how this would work with your display.

**Reply by RaspCla** [reply]
ok, I already feared it will not work, do not know how it was done by Christian. Maybe he had a controller in between.

If there is any idea how I could use this display with Arduino or RaspberryPi without spending to much effort/money please let me know.

Thanks a lot for support

**Reply by olikraus** [reply]
Recently we added support to a HD44102 display (https://github.com/olikraus/u8g2/issues/1492). Probably also some kind of outdated technology, but at least the HD44102 is a kind of controller, which can be handled by u8g2.
It was nice to reactivate such old technology, but the effort was limited because we had been able to build on the existing u8g2 communication features (8 bit parallel interface). This is different with your display: We would need a complete new communication interface  for that display.

What I wanted to say: If you need a display for a project, then better look for a more modern LCD module. Still crosscheck whether it is supported by U8g2 (or any other Arduino lib). Searching for "lcd module st7565" may already reveal a lot of inexpensive LCDs.

**Reply by RaspCla** [reply]
> 
> What I wanted to say: .....
It's just the way arround. I own several of these displays and questioned myselft whether there is already a solution/implementation available. If not: no problem, typically I buy the stuff I need and as you said, displays are no more expensive.
